terraform/groups/ecs-service/profiles/development-eu-west-2/cidev/vars

```diff
@@ -2,14 +2,6 @@ environment = "cidev"
 aws_profile = "development-eu-west-2"
 
 log_level = "debug"
-chs_url = "https://cidev.aws.chdev.org"
-piwik_url = "https://matomo.platform.aws.chdev.org/"
-piwik_site_id = "7"
 cdn_host = "d3uvya5a8a1ncx.cloudfront.net"
-human_log = "1"
 cookie_domain = "cidev.aws.chdev.org"
-cookie_expiration_in_seconds = "3600"
-download_filename_prefix = "CH_SO_OBJ_"
-max_file_size_bytes = "4194304"
-show_service_offline_page = "false"
-api_url = "https://api.cidev.aws.chdev.org"
+cookie_expiration_in_seconds = "3600"
```

Comment:
I don't think this var is used by transaction-search-tool. I believe the used config is default_session_expiration:

https://github.com/companieshouse/transaction-search-tool/blob/main/src/config.ts#L31